### PR TITLE
HoC Page Refresh

### PR DESCRIPTION
**Pull request by adamkecskes**
HoC page refreshed:
 - Header text changed
 - Webinar banner added
 - Activities split to featured (top) and secondary (bottom)
 - Video links added that opens&plays youtube video in popup modal

The old Webinar Banner was replaced by HoC Banner on classroom pages both (ozaria&coco) 

hoc

HoC Banner on Ozaria:
ozaria

HoC Banner on CoCo:
Saját_osztályok___CodeCombat

### CODE REVIEW

File: ozaria/site/components/teacher-dashboard/BaseMyClasses/index.vue [hunk 1]
```diff
@@ -128,8 +128,7 @@
 
 <template>
   <div>
-    <banner-webinar />
-
+    <banner-ho-c />
```
Comment (nwinter) [inline ozaria/site/components/teacher-dashboard/BaseMyClasses/index.vue:131]:
```suggestion
    <banner-hoc />
```

File: app/views/special_event/HoC2018Component.vue [hunk 2]
```diff
@@ -55,29 +55,45 @@
...
+            .row
+              .col-md-4
+                img.activity-tile(src='/images/pages/teachers/hour-of-code/goblins-n-glory.png' alt="")
+              .col-md-8
+                h3(data-i18n="hoc_2018.featured_title_1")
+                h6(data-i18n="hoc_2018.featured_subtitle_1")
+                p(data-i18n="hoc_2018.featured_blurb_1")
+                div
+                  a.btn.btn-primary.btn-md(href="/play/goblins-hoc?hour_of_code=true" data-i18n="hoc_2018.try_activity")
+                  button.btn.btn-primary.btn-md(data-toggle="modal" data-video-id="s0RI4IXKE_o" data-target="#youtubeModal" @click="playVideo" data-i18n="hoc_2018.play_video")
```
Comment (nwinter) [inline app/views/special_event/HoC2018Component.vue:73]: Test video alert ;)
```suggestion
                  button.btn.btn-primary.btn-md(data-toggle="modal" data-video-id="YZjnpqb73ms" data-target="#youtubeModal" @click="playVideo" data-i18n="hoc_2018.play_video")
```

**Review by nwinter — CHANGES_REQUESTED** [review]
Reviewed from code/screenshots. Love it, just left some recommendations for styling changes and a couple typos/tweaks.

File: app/templates/courses/teacher-classes-view.coco.pug [hunk 3]
```diff
@@ -62,7 +62,7 @@ block content
               a.see-all-office-hours.btn.btn-primary.btn-small.pull-right(data-i18n="general.more") More
 
     if !me.hideOtherProductCTAs()
-      .banner-webinar
+      .banner-hoc
```
[the remark this answers is not in the document]
Comment (adamkecskes) [reply app/templates/courses/teacher-classes-view.coco.pug:65]: Absolutely agree. Will do this way next time.

File: ozaria/site/components/teacher-dashboard/BaseMyClasses/index.vue (hunk 1 above)
Comment (adamkecskes) [reply ozaria/site/components/teacher-dashboard/BaseMyClasses/index.vue:131]: It is actually intentional. It's auto generated from `BannerHoC` which results in `banner-ho-c`.

**Pull request merged**